### PR TITLE
openshift docs based on new k8s docs

### PR DESCRIPTION
**Pull request by gggina**

### CODE REVIEW

File: stackpacks/integrations/openshift.md [hunk 1]
```diff
@@ -1,63 +1,307 @@
...
+For example:
+
+```
+helm install \
+--namespace stackstate \
+--create-namespace \
+--set-string 'stackstate.apiKey'='<your-api-key>' \
+--set-string 'stackstate.cluster.name'='<your-cluster-name>' \
+--set-string 'stackstate.cluster.authToken'='<your-cluster-token>' \
+--set-string 'stackstate.url'='<your-stackstate-url>' \
```
Comment (rb3ckers) [inline stackpacks/integrations/openshift.md:84]: We need to add least include this setting here as well:
`--set-string 'agent.scc.enabled'='true' \`

I'm not sure if that is all though. @hierynomus you tested this I believe. Is this all customization that is needed for the agent?

File: stackpacks/integrations/kubernetes.md [hunk 2]
```diff
@@ -214,7 +214,7 @@ For further details, refer to the [Kubernetes API documentation \(kubernetes.io\
 
 ### Component actions
 
-A number of [actions](use/views/topology-perspective.md#actions) are added to StackState when the Kubernetes StackPack is installed. They are available from the **Actions** section on the right of the screen when a Kubernetes component is selected or from the component context menu, displayed when you hover over a Kubernetes component in the topology perspective
+A number of [actions](/use/views/topology-perspective.md#actions) are added to StackState when the Kubernetes StackPack is installed. They are available from the **Actions** section on the right of the screen when a Kubernetes component is selected or from the component context menu, displayed when you hover over a Kubernetes component in the topology perspective
```
Comment (rb3ckers) [inline stackpacks/integrations/kubernetes.md:217]: Some more references to Kubernetes here that should be Openshift

File: stackpacks/integrations/openshift.md [hunk 3]
```diff
@@ -1,63 +1,307 @@
...
+## Setup
 
 ### Prerequisites
 
-The following prerequisites are required for manual installation:
+The following prerequisites are required to install the Openshift StackPack and deploy the StackState Agent and Cluster Agent:
+
+* An Openshift Cluster must be up and running.
+* A recent version of Helm 3.
+* A user with permissions to create privileged pods, ClusterRoles and ClusterRoleBindings:
```
Comment (rb3ckers) [inline stackpacks/integrations/openshift.md:41]: And SCC's (Security Context Constraints), needed to have OpenShift allow the agent privileged access

File: stackpacks/integrations/kubernetes.md (hunk 2 above)
Comment (hierynomus) [reply stackpacks/integrations/kubernetes.md:217]: This is the kubernetes page I think @remco

File: stackpacks/integrations/openshift.md [hunk 4]
```diff
@@ -1,63 +1,307 @@
...
-* Reporting the different type of workloads.
-* Reporting nodes, pods, containers and services.
+The Openshift integration collects topology data in an Openshift cluster as well as metrics and events.
 
-## Installation
+- StackState Agent V2 is deployed as a DaemonSet with one instance **on each node** in the Openshift cluster:
+    * Host information is retrieved from the Openshift API
+    * Container information is collected from the Docker daemon
+    * Metrics are retrieved from kubelet running on the node and also from kube-state-metrics if this is deployed on the same node
+- StackState Cluster Agent is deployed with a Deployment. There is one instance for the entire Openshift cluster:
```
Comment (hierynomus) [inline stackpacks/integrations/openshift.md:23]: deployed *as* a Deployment

File: stackpacks/integrations/openshift.md (hunk 1 above)
Comment (hierynomus) [reply stackpacks/integrations/openshift.md:84]: For the agent this is the only addition as far as I'm aware.

File: stackpacks/integrations/openshift.md [hunk 5]
```diff
@@ -1,63 +1,307 @@
...
+- Namespace → CronJob, DaemonSet, Deployment, Job, ReplicaSet, Service, StatefulSet
+- Node → Cluster relation
+- Pod → ConfigMap, Container, Deployment, Node, Secret, Volume
+- ReplicaSet →  Pod
+- Service → ExternalService, Pod
+- StatefulSet → Pod
+- Direct communication between processes
+- Process → Process communication via Openshift service
+- Process → Process communication via headless Openshift service
+
```
Comment (hierynomus) [inline stackpacks/integrations/openshift.md:174]: - Container -> PersistentVolume
- Pod -> PersistentVolume

File: stackpacks/integrations/openshift.md (hunk 1 above)
Comment (gggina) [reply stackpacks/integrations/openshift.md:84]: updated

File: stackpacks/integrations/kubernetes.md (hunk 2 above)
Comment (gggina) [reply stackpacks/integrations/kubernetes.md:217]: yeah, there was an error in the link on the k8s page that I noticed when I copied it.

File: stackpacks/integrations/openshift.md (hunk 4 above)
Comment (gggina) [reply stackpacks/integrations/openshift.md:23]: updated

File: stackpacks/integrations/openshift.md (hunk 3 above)
Comment (gggina) [reply stackpacks/integrations/openshift.md:41]: updated

File: stackpacks/integrations/openshift.md (hunk 5 above)
Comment (gggina) [reply stackpacks/integrations/openshift.md:174]: updated

**Pull request merged**